bare-armv7emhf-nano_newlib/Dockerfile.in

```diff
@@ -5,8 +5,8 @@ LABEL maintainer="Chen Tao t.clydechen@gmail.com"
 
 # This is for armv7e-m+fp bare metal
 
-# Crosstool-ng version 1.25.0
-ENV CT_VERSION 8fa98eeeff9bc53478d97ef722f366fea151ae64
```

Comment:
I don't know why this was previously using the hash, I thought these were all changed in #697?
@bensuperpc was this missed, or intentionally left?

Reply:
I think it's an oversight ^^